README.md

```diff
@@ -31,10 +31,13 @@ This project tests all builds against each supported version of Fedora, the late
 
 ## Out of scope
 
-* Signing and pushing images to various image storages.
```

Comment:
Should we leave this, with a note that Podman does it, but we still recommend Skopeo for serious work? Our sign/verify code isn't as mature as theirs